### PR TITLE
Step4: implement post and show comments

### PR DESCRIPTION
**Pull request by shaynemei**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -27,3 +27,25 @@ function addRandomGreeting() {
...
 }
+
+async function getRandomQuote() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('quote-container').innerText = quote;
+}
+
+function getComments() {
+  fetch('/data').then(response => response.json()).then((history) => {
```
Comment (labib666) [inline portfolio/src/main/webapp/script.js]: You may want to split the servlet into two and name them properly.
Creating single endpoint for all your tasks does not scale well.

**Review by labib666 — CHANGES_REQUESTED** [review]
Some changes will be helpful for readability.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (shaynemei) [reply portfolio/src/main/webapp/script.js]: Do you mean creating another servlet.java to handle querying comments from DataStore?

Comment (labib666) [reply portfolio/src/main/webapp/script.js]: Yes. Of course it will have a different name.

**Comment by shaynemei** [comment]
pushed changes to PR #8

**Pull request closed without merging**